css/style.css

```diff
@@ -851,7 +850,7 @@ a.top-scroll:hover {
 .img-section img{
     display: block;
 }
-
+/* Some of these seem broken? */
```

Comment:
@ArshKochhar Some of the media queries seem to have unclosed curly braces. Could you double check if they're good?